CHANGELOG/v1.12.2.md

```diff
@@ -0,0 +1,31 @@
+## Changes by Kind
+
+### Bug or Regression
+
+- ASO environment variables can now be set to refer to a non-public (e.g. government) cloud instance and set ASO's periodic resync interval: https://azure.github.io/azure-service-operator/guide/aso-controller-settings-options/ ([#4411](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4411), [@k8s-infra-cherrypick-robot](https://github.com/k8s-infra-cherrypick-robot))
```

Comment:
can we update the authors to the original commit authors for these cherry pick commits?